tests/type-annotations/main.fmf

```diff
@@ -0,0 +1,53 @@
+summary: Check type annotations in tmt code
+duration: 10m
+environment:
+    LANG: en_US.UTF-8
+
+    WITH_SYSTEM_PACKAGES: "no"
+
+tier: 0
+
+adjust+:
+  - when: distro == centos-stream-9
```

Comment:
This was fixed on rhel-9 btw. Do we need to keep compatibility with older releases, or can we get rid of the workaround completely?